scripts/spack/specs.json

```diff
@@ -21,8 +21,8 @@
       "intel@19.0.4~openmp+devtools+mfem+c2c" ],
 
     "toss_4_x86_64_ib":
-    [ "gcc@10.3.1+devtools+hdf5+mfem+c2c",
-      "clang@14.0.6+devtools+hdf5+mfem+c2c",
+    [ "gcc@10.3.1+devtools+hdf5+mfem+c2c+scr",
+      "clang@14.0.6+devtools+hdf5+mfem+c2c+scr",
       "intel@2022.1.0+devtools+hdf5+mfem+c2c" ],
```

Comment:
`+scr` was added based on the toss3 specs: https://github.com/LLNL/axom/blob/c35431a1ee71a7c46fe54805e8aa5278120d613e/scripts/spack/specs.json#L16-L21

On another note, are there any machines still on toss3? Should we keep or deprecate these toss 3 specs?

Reply:
There are no longer toss3 machines.

Reply:
Toss 3 cleanup issue opened here: https://github.com/LLNL/axom/issues/1208